skyfield/elementslib.py

```diff
@@ -9,17 +9,28 @@
                    sinh, sqrt, arccos, arctanh, zeros_like, ones_like, divide,
                    where, pi, cross)
 
-def osculating_elements_of(position, reference_frame=None):
+def osculating_elements_of(position, reference_frame=None, gm_km3_s2=None):
     """Produce the osculating orbital elements for a position.
 
-    The ``position`` should be an :class:`~skyfield.positionlib.ICRF`
-    instance like that returned by the ``at()`` method of any Solar
-    System body, specifying a position, a velocity, and a time.  An
-    instance of :class:`~skyfield.elementslib.OsculatingElements` is
-    returned.
+    Parameters:
```

Comment:
The docstring needs to be valid Sphinx, so the errors here probably come from the erratic shifts in indentation. I simply prefer to write normal paragraphs with each line indented the same — take a look at the existing docstring here for something that Sphinx likes. Blank lines separate paragraphs.